packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 ### Fixed
 #### salesforcedx-vscode-apex
 
-* Improve syntax highlighting for built-in Apex classes, methods, and types ([PR #474](https://github.com/forcedotcom/salesforcedx-vscode/pull/474))
+* Improve syntax highlighting for built-in Apex classes, methods, and types ([PR #474](https://github.com/forcedotcom/salesforcedx-vscode/pull/474), [PR #484](https://github.com/forcedotcom/salesforcedx-vscode/pull/484))
```

Comment:
Should we add another bullet point to highlight the improvements on soql syntax highlighting and for loops ?

Reply:
I think it's fine. I was lumping it as highlighting for Apex clases, methods, and types.

Reply:
:shipit: